_examples/rss_reader.go

```diff
@@ -0,0 +1,50 @@
+package main
+
+import (
+	"fmt"
+	"log"
+
+	"github.com/jroimartin/gocui"
+
+	"github.com/crazcalm/go-rss-reader/database"
+	"github.com/crazcalm/go-rss-reader"
+	"github.com/crazcalm/go-rss-reader/cli"
+)
+
+func main() {
+	cli.GlobalConfig = cli.NewConfig()
+	config := cli.GlobalConfig
+	err := config.CliParse()
+	if err != nil {
+		log.Panicln(err)
```

Comment:
In the future, I will completely remove this log, but, for now, Keep this and make thi a Fatal log.